standard-controller/src/main/java/io/enmasse/controller/standard/AddressProvisioner.java

```diff
@@ -493,4 +481,22 @@ private BrokerInfo(String brokerId, double credit) {
         }
 
     }
+
+    public static class BrokerIdGenerator {
+        private final Random random = new SecureRandom();
```

Comment:
Does this need to be a SecureRandom or just a Random?  I don't know if it is still an issue but using too many SecureRandoms used to cause pauses while the JVM waited for entropy

Reply:
Fixed